src/main/java/org/apache/sling/jcr/resource/internal/helper/jcr/JcrResourceProvider.java

```diff
@@ -478,6 +479,44 @@ public Resource create(final @NotNull ResolveContext<JcrProviderState> ctx, fina
         }
     }
 
+    @Override
+    public boolean orderBefore(@NotNull ResolveContext<JcrProviderState> ctx, @NotNull Resource parent, @NotNull String name,
+            @Nullable String followingSiblingName) throws PersistenceException {
+        Node node = parent.adaptTo(Node.class);
+        try {
+            if (node == null) {
```

Comment:
This could be moved outside the try/catch block.

Reply:
Done in https://github.com/apache/sling-org-apache-sling-jcr-resource/pull/18/commits/d74a24df6297e7a78acd2127d19229a19752f5a4.